src/imio/smartweb/core/tests/test_rest.py

```
@@ -804,94 +807,142 @@ def test_enrich_response(self, mock_view_url, mock_request):
+        mock_request.return_value = fake
+
+        service = self.traverse(
+            "/plone/@events_request_forwarder/belleville/abc123def456abc123def456abc12345"
         )
-        # non-UID @id segment (entity/folder) left untouched
-        self.assertNotIn("smartweb_url", response["items"][1])
+        service.reply()
 
+        # non-listing events GET → forwarded as-is, no delegation to the view
+        mock_view.assert_not_called()
+        args, kwargs = mock_request.call_args
```

Ruff (0.15.21)

[warning] Unpacked variable `kwargs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)